src/interfaces/profile.interface.ts

```diff
@@ -1,11 +1,20 @@
 export interface Profile {
     name: string;
     team: string;
+    type: ProfileType;
     apiToken: string;
     authenticationType: AuthenticationType;
+    clientId?: string;
+    clientSecret?: string;
+    scopes?: string[];
+    clientAuthenticationMethod?: ClientAuthenticationMethod;
+    refreshToken?: string;
+    expiresAt?: number;
 }
 
 export type AuthenticationType = "Bearer" | "AppKey";
+export type ProfileType = "Device Code" | "Client Credentials" | "Key";
+export type ClientAuthenticationMethod = "client_secret_basic" | "client_secret_post";
```

Comment:
Can we also export some constants with these values? I saw we are using these strings in a few places.

Reply:
[edbcae3](https://github.com/celonis/content-cli/pull/163/commits/edbcae3efce6020983d4cb0290b371a1fc1f613b)